includes/wc-product-functions.php

```diff
@@ -84,7 +84,7 @@ function wc_get_products( $args ) {
 	 * Generate WP_Query args.
 	 */
 	$wp_query_args = array(
-		'post_type'      => 'variation' === $args['type'] ? 'product_variation' : 'product',
+		'post_type'      => array( 'product', 'product_variation' ),
```

Comment:
@mikejolley I changed it back so you can get both products and product variations in one go -- but maybe that's not something we want.